### ISSUE TITLE
The definition is ignored in case of HTMLPurifier_Config::inherit() usage

### ISSUE THREAD

**Issue by xgrn** [xemlock/htmlpurifier-html5#16]
Hi.

Thanks for the great library which saves developer lifetime :smile: 

I'm using this config to override the Purifier configuration in a Symfony project with the exercise/htmlpurifier-bundle

The bundle creates the parent configuration than uses HTMLPurifier_Config::inherit() method to create the child one. The method implementation is taken from the parent class, not HTML5Config as the following.
```php
    /**
     * Creates a new config object that inherits from a previous one.
     * @param HTMLPurifier_Config $config Configuration object to inherit from.
     * @return HTMLPurifier_Config object with $config as its parent.
     */
    public static function inherit(HTMLPurifier_Config $config)
    {
        return new HTMLPurifier_Config($config->def, $config->plist);
    }
```
As a result all the child configurations are HTMLPurifier_Config  instances instead of HTMLPurifier_HTML5Config which causes errors as they don't support HTML5 tags.

I'm using a workaround inheriting the base class like:
```php
class HTMLPurifier_AltHTML5Config extends \HTMLPurifier_HTML5Config
{
    public static function inherit(HTMLPurifier_Config $config)
    {
        return new static($config->def, $config->plist);
    }
}
```

But the 'inherit()' method should be overridden as well, I suppose.

Thanks again.
Best wishes.

**Comment by xemlock** [comment]
@xgrn Yes, you're right. I'll try to provide a fix for this during weekend. Thanks for reporting 👍

**Comment by xgrn** [comment]
Looking forward. Thanks!

**Comment by xemlock** [comment]
@xgrn Please let me know if the recently released [v0.1.7](https://github.com/xemlock/htmlpurifier-html5/releases/tag/v0.1.7) solves your issue.

**Comment by xgrn** [comment]
Hi. It works like a charm after update. Many thanks!

Sorry for the silence: I was badly sick and couldn't test.
